Makefile

```diff
@@ -18,13 +18,15 @@ default: help
 
 VERSION ?= 1.6.0
 
-# 2.15 image: "2.15.0-alpine"
+
 TARGET_APISIX_VERSION ?= "3.1.0-centos"
 APISIX_ADMIN_API_VERSION ?= "v3"
-ifneq ($(APISIX_ADMIN_API_VERSION), "v3")
-ifeq ($(TARGET_APISIX_VERSION), "3.1.0-centos")
-	TARGET_APISIX_VERSION = "2.15.0-alpine"
-endif
+ifeq ($(APISIX_ADMIN_API_VERSION), "v3")
+	TARGET_APISIX_VERSION ?= "3.2.0-centos"
+else ifeq ($(APISIX_ADMIN_API_VERSION), "v2")
+	TARGET_APISIX_VERSION ?= "2.15.3-centos"
+else ifeq ($(APISIX_ADMIN_API_VERSION), "dev")
+	TARGET_APISIX_VERSION ?= "dev"
```

Comment:
`APISIX_ADMIN_API_VERSION` only `v2` and `v3`, no `dev`.
https://apisix.apache.org/docs/apisix/admin-api/#v3-new-feature
Why modify the logic here? Can't just modify the image tag?